zkevm-circuits/src/copy_circuit.rs

```diff
@@ -110,7 +112,7 @@ impl<F: Field> SubCircuitConfig<F> for CopyCircuitConfig<F> {
             bytecode_table,
             copy_table,
             q_enable,
-            randomness,
+            challenges,
         }: Self::ConfigArgs,
     ) -> Self {
         let q_step = meta.complex_selector();
```

Comment:
When the dest is `CopyDataType::RlcAcc`, the write step contains an RLC, so `value` should also be `SecondPhase`.
See https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/56642adc9e9206ef38e3e4dc141192f0fce5eba0/zkevm-circuits/src/table.rs#L990-L996